src/style.css

```
@@ -0,0 +1,94 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  --snake__size: 300px;
```

```suggestion
  --cell__size: 300px;
```
it is not the whole snake size 😉
and you may move it to `:root{}`